packages/vitest/package.json

```diff
@@ -152,7 +151,7 @@
     "strip-literal": "^1.0.1",
     "tinybench": "^2.5.0",
     "tinypool": "^0.8.1",
-    "vite": "^3.1.0 || ^4.0.0 || ^5.0.0-0",
+    "vite": "^5.0.0-0",
```

Comment:
This will break with beta<15. Let's have it as `^5.0.0-beta.15 || ^5.0.0` until 5.0.0 is released

Reply:
Yeah, I think you're right